dotcom-rendering/src/components/MainMedia.tsx

```diff
@@ -33,6 +33,10 @@ const noGutters = css`
 	}
 `;
 
+const padBottom = css`
+	padding-bottom: 6px;
```

Comment:
I think we try to only use source spacing which is multiples of 4, maybe we can use 4px here and then use [space[1]](https://guardian.github.io/storybooks/?path=/docs/source-foundations_space--docs)?